src/ui/settingsdialog.cpp

```diff
@@ -68,6 +68,24 @@ const char DocsetNameProperty[] = "docsetName";
 const char DownloadTypeProperty[] = "downloadType";
 const char DownloadPreviousReceived[] = "downloadPreviousReceived";
 const char ListItemIndexProperty[] = "listItem";
+
+QString timeSinceLastUpdate(const QDateTime &updateTime)
+{
+    int diff = updateTime.secsTo(QDateTime::currentDateTime());
+
+    if (diff <= 5)      return QStringLiteral("just now");
+    if (diff <= 20)     return QStringLiteral("%1 seconds ago").arg(diff);
```

Comment:
The [article](http://doc.qt.io/archives/qq/qq19-plurals.html) linked in the above documentation page has more information.